packages/legend-query/src/components/QueryBuilderExplorerPanel.tsx

```diff
@@ -520,7 +527,18 @@ const QueryBuilderExplorerTreeNodeContainer = observer(
                   </button>
                 )}
                 <QueryBuilderPropertyInfoTooltip
```

Comment:
Lets create another tooltip component for subclass. we should never create a property of a subclass